src/main/java/org/jabref/gui/undo/RedoAction.java

```diff
@@ -34,7 +46,8 @@ public void execute() {
         try {
             libraryTab.getUndoManager().redo();
             dialogService.notify(Localization.lang("Redo"));
-        } catch (CannotRedoException ex) {
+        } catch (
+                CannotRedoException ex) {
```

Comment:
Please also fix the catch indent (not your fault, IntelliJ does this by default when we format the document).